types/runtime.ts

```diff
@@ -62,13 +62,22 @@ export type StorageConfig = {
   storageEvent?: "finalize" | "delete" | "archive" | "metadataUpdate";
 } & FunctionConfig;
 
+export type WebhookConfig = {
+  type: "webhook";
+  webhook: {
+    type: "bigcommerce";
```

Comment:
will this always be bigcommerce?

Reply:
It could be extended to other webhook types in future. But for now its only bigcommerce